vignettes/generic.Rmd

```diff
@@ -313,7 +313,7 @@ ae_02 <- ae_01 %>%
     dataset_add = ae_01,
     filter_add = AESTDY > 0,
     by_vars = exprs(USUBJID),
-    order = exprs(as.integer(factor(AESEV, levels = c("SEVERE", "MODERATE", "MILD")))),
+    order = exprs(AESEV = as.integer(factor(AESEV, levels = c("SEVERE", "MODERATE", "MILD")))),
```

Comment:
This changes the output dataset. The bug fix for `derive_vars_joined()` should not require to change the calls to `derive_vars_joined()`.